lib/lightstep/tracer.rb

```diff
@@ -28,20 +28,16 @@ class ConfigurationError < LightStep::Tracer::Error; end
     # @param access_token [String] The project access token when pushing to LightStep
     # @param transport [LightStep::Transport] How the data should be transported
     # @param tags [Hash] Tracer-level tags
-    # @param propagator [Propagator] Symbol one of :lightstep, :b3 indicating the propgator
-    #   to use
     # @return LightStep::Tracer
     # @raise LightStep::ConfigurationError if the group name or access token is not a valid string.
     def initialize(component_name:,
                    access_token: nil,
                    transport: nil,
-                   tags: {},
-                   propagator: :lightstep)
+                   tags: {})
```

Comment:
Might make sense to either add the new propagator option (even if only the single option exists), **or** to take out the propagator code _completely_?

Reply:
I'm fine leaving the option as long as it's not confusing. Removing the code completely is well, somewhat of a challenge. This repo has squash and merge setup and we need some code sandwiched into one of the B3 commits that fixed CI. I wanted a single commit that would be easy to "undo". I can add the B3 work to the unreleased section so it's not too surprising as you suggest below. Does that work?